### PR TITLE
add another analysis timing to the dashboard

### PR DESCRIPTION
**Pull request by devoncarew**
- add another analysis timing to the dashboard
- add a favicon for flutter
- slightly cheaper clones of the repo (we may want to think about keeping the flutter repo around between runs, and using `git pull` to update)
- don't send analytics from the bot

@yjbanov

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -46,12 +46,14 @@ mkdir -p $DATA_DIRECTORY
 # --------------------------
 
 cd $ROOT_DIRECTORY
+# TODO(devoncarew): Do we need to blow this away each time?
```
Comment (yjbanov) [inline run.sh]: I think so. This step doesn't take too long. We are running examples from inside the repo, and our tools are doing all sorts of caching, incremental building, and so on. A bug there could cause us to run wrong code and could be very hard to debug.

File: run.sh [hunk 2]
```diff
@@ -46,12 +46,14 @@ mkdir -p $DATA_DIRECTORY
...
 rm -rf $FLUTTER_DIRECTORY
-git clone https://github.com/flutter/flutter.git
+git clone --depth 1 https://github.com/flutter/flutter.git
```
Comment (yjbanov) [inline run.sh]: Good call!

**Comment by yjbanov** [comment]
LGTM

File: run.sh (hunk 1 above)
Comment (devoncarew) [inline run.sh]: I was only a slow network for a bit yesterday, and cloning the repo, downloading the dart sdk, and then all the engine deps, took 10-15 mins. That's probably not a common use case, but there may be some low-risk way of preserving some of the checkout.

I'll remove this TODO: for now.

**Comment by devoncarew** [comment]
Ready to go! (I can't merge the PR)

**Pull request merged**